jekyll/_cci2_ja/index.md

```diff
@@ -2,7 +2,9 @@
 layout: classic-docs
 title: "CircleCI ドキュメントへようこそ"
 description: "CircleCI ドキュメントへようこそ"
-permalink: /ja/2.0/
+permalink: /ja/
+redirect_from:
+        - /ja/2.0/
```

Comment:
same question:

```suggestion
redirect_from: /ja/2.0/
```

Reply:
updated thanks! yes this works as per the documentation [link](https://github.com/jekyll/jekyll-redirect-from)